README.md

```diff
@@ -1,21 +1,38 @@
 # Ably Spaces
 
-**This is an experimental API, not meant for production usage.**
+This respository consists an API for managing and accessing Ably Spaces. Please note that this is an experimental API. You can try it out by following the usage instructions below.
 
-If you have any feedback or are interested in what we are doing please [contact us](https://ably.com/contact).
+If you have any feedback or are interested in what we are doing please reach out to us at [beta@ably.com](mailto:beta@ably.com).
 
-API for managing and accessing Ably Spaces.
+## Concepts
+The Ably Spaces library provides purpose-built methods and properties to enable multiplayer collaboration in any web application. You can set up a collaborative "space" and subscribe to different kind of updates to see who else is in the space with you, their live location within the app and any updates they are making.
 
-**Class definitions are available [here](/docs/class-definitions.md).**
+This library is built as an extention to our [existing JS SDK](https://github.com/ably/ably-js), so you'll need to an Ably JS client to be able to use this library. The usage instructions will help you set that up.
```

Comment:
typo - "extension"
Missing word(s) - maybe "You'll need to be able to use an Ably JS Client"?